Automattic/wp-calypso
# Untangling: add Marketing -> Marketing Tools subtab

Related to:

- https://github.com/Automattic/dotcom-forge/issues/9608

## Proposed Changes

This PR adds a new Marketing -> Marketing Tools subtab on the site management panel, under the `untangling/hosting-menu` flag. It shows what we're currently showing on Tools/Hosting -> Marketing.

|Tools/Hosting -> Marketing<br>`/marketing/tools/:site`|new Marketing tab<br>`/sites/marketing/tools/:site`|
|-|-|
|<img width="839" alt="image" src="https://github.com/user-attachments/assets/6e98e33c-3c94-47a9-9f0d-92dd5965b6f3">|<img width="986" alt="image" src="https://github.com/user-attachments/assets/a291a98c-f4db-4564-8299-b61ce0470492">|

Note that we're not migrating Business Tools; it's being removed here:

- https://github.com/Automattic/wp-calypso/pull/95848

## Why are these changes being made?


pbxlJb-6ye-p2

## Testing Instructions



- Go to `/sites/marketing/tools/:site`; verify the new Marketing Tools page is working as expected.
- Go to `/overview/:site?flags=-untangling/hosting-menu`; verify the new Marketing tab is NOT shown.
- Go to `/marketing/tools/:site`; verify the old Marketing Tools page is still working as expected.
- Go to `/marketing/tools/:site?flags=-untangling/hosting-menu`; verify the old Marketing Tools page is still working as expected.

## Pre-merge Checklist
